netbox/templates/extras/script.html

```diff
@@ -47,16 +47,44 @@
           {% csrf_token %}
           <div class="field-group my-4">
             {% if form.requires_input %}
-              <div class="row mb-2">
-                <h5 class="offset-sm-3">Script Data</h5>
-              </div>
+              {% if script.Meta.fieldsets %}
+                {# Render grouped fields according to declared fieldsets #}
+                {% for group, fields in script.Meta.fieldsets %}
+                  <div class="field-group mb-5">
+                    <div class="row mb-2">
+                      <h5 class="offset-sm-3">
+                        {% if group %}
+                          {{ group }}
+                        {% else %}
+                          {{ model|meta:"verbose_name"|bettertitle }}
+                        {% endif %}
+                      </h5>
+                    </div>
+                    {% for name in fields %}
+                      {% with field=form|getfield:name %}
+                        {% if field.name in form.nullable_fields %}
```

Comment:
I don't believe we support `nullable_fields` for script forms; IIRC this is limited to bulk edit forms.

Reply:
Good catch.  I will resolve this.